src/Core/System/Snippet/Command/ValidateSnippetsCommand.php

```diff
@@ -16,6 +16,9 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\Question;
 
+/**
+ * @phpstan-type Snippets array<string, string|mixed>
```

Comment:
```suggestion
 * @phpstan-type Snippets array<string, string|mixed>
```

saying the value is `string` or `value` is quite useless :slightly_smiling_face: if it could be `mixed`, then there is no need to explicitly state `string`.

but better would be to specify exactly what the value could be instead of just saying `mixed` (meaning everything :grin:)

Reply:
At first I tried a recursive type, but it seems PHPstant didn't like that :/ Hope `array<string, mixed>` is better for the inner ´mixed`

Reply:
structs would solve that problems :grin: